miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/TileTag.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2002-2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.bbsrc.tgac.miso.webapp.util.form;
+
+import java.util.Collection;
+
+import org.springframework.web.servlet.tags.RequestContextAwareTag;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@SuppressWarnings("serial")
+public class TileTag extends RequestContextAwareTag {
+  private boolean alwaysShow;
+
+  private Object items;
+
+  private String target;
+
+  @Override
+  protected int doStartTagInternal() throws Exception {
+    if (items == null) {
+      return SKIP_BODY;
+    }
+    @SuppressWarnings("unchecked")
+    Collection<Object> items = (Collection<Object>) this.items;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "items" which hides the field declared at line 29. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)